include/nuttx/lin.h

```diff
@@ -44,45 +44,43 @@
 #define LIN_RTR_FLAG              CAN_RTR_FLAG  /* Describe the direction of sending and receiving */
 #define LIN_ERR_FLAG              CAN_ERR_FLAG  /* The flag indicate  this is LIN err_frame */
 
-/* Bit flags on can_frame.types
- *
- * Use can_frame.types to identify other special frame.
- *
- * LIN_TCF_FLAG : TxConfirmation message frame. Lower_half use this flag to
- *                confirm frame-transmit
- *
- * LIN_EVT_FLAG : Event message frame. Lower_half use this flag to report
- *                state switch event
- *
- * LIN_CACHE_RESP_FLAG: When slave response to master, slave node should send
- *                      frame immediately which already be cached in last
- *                      transmission in case of response interval over time.
- *
- * LIN_CHKSUM_EXT_FLAG: LIN checksum have two types, default type will be
- *                      classic checksum
- *
- * LIN_SINGLE_RESP_FLAG: Cache LIN frame only work once, then will be cleared
+/* When slave response to master, slave node should send  frame immediately
```

Comment:
```suggestion
/* When slave responds to master, slave node should immediately send frame
```

Reply:
CAN_EVT_FLAG  uses bit29.The CAN ID of the extended frame format (EFF) is 29 bits, ranging from 0 to 0x1FFFFFFF.CAN_EVT_FLAG is not good for the Extended frame format of can.
In CAN, CAN_EVT_FLAG is not currently used.Therefore, this identification bit was deleted in CAN.


LIN's ID only occupies 6 bits.Tlin has more usable bits, so consider a special FLAG that can not be shared with can.
LIN_EVT_FLAG of LIN is in use.Therefore, the following definition was added
'''
#define LIN_EVT_FLAG              (1 << (LIN_ID_BITS + 3))
'''

Reply:
What's the meaning？

Reply:
Do you want me to change these comments to the previous ones?